### DISCUSSION TITLE
Add the ability to select browser on opening url.

### DISCUSSION THREAD

**Discussion by jasonistre** [nref/BrowseRouter#23] [Ideas]
It would be nice to simply have a popup when I click any url outside of a browser that pops up a simple window that lets me select either edge or firefox everytime.

**Comment by nref** [comment]
Thanks for the suggestion! I agree this and other UI elements would be nice. Unfortunately, I don't have much time to add it. In the meantime, you might try BrowserTamer which has a slick UI. https://github.com/aloneguid/bt

**Comment by nref** [comment]
My paid app Linklever now opens a window to select a browser. https://linklever.net/blog/2025/10/browser-picker/
